pages/workspace/tracking-plan/properties.mdx

```diff
@@ -112,3 +112,78 @@ For configuring per-event and per-source you can change the overall setting for
     alt="Image shows how to change property presence between events"
   />
 </center>
+
+### <a name="snowplow-objects"></a> Snowplow Objects
+
+To set up a Snowplow object, the property has to be attached to an event that is being sent to Snowplow. This can be configured in the source configuration of an event. Then you can create a property and set it's type to an object. When you do that, then the option appears on the property to make it a Snowplow context. When you check that box, the code will be generated such that the object is converted to a Snowplow object.
+
+<center>
+  <img
+    src={require('../../../images/workspace/properties/snowplow-context-example.png')}
+    width="60%"
+    alt="Example of a Snowplow context definition in Avo"
+  />
+</center>
+
+Here's how an individual Snowplow context is translated from what is defined in the UI to the context data structure:
+
+```json
+{
+  "data": {
```

Comment:
One more thing that every Snowplow context and the root event sent to Snowplow need to have is the pinned "schema" property, i.e. this is how it's done in BoF case https://www.avo.app/schemas/gsaQteWxmXjpUc8AGauc/events/2bUG-rWa5C